libfbc/src/fbc.c

```diff
@@ -23,32 +23,34 @@ static void _fbcTask(void* param) {
 
 static int _prev = 0;
 static unsigned int _count = 0;
-bool fbcStallDetect(fbc_t* fbc) {
-	unsigned int min_stuck = fbc->acceptableTolerance >> 3;
-	if (min_stuck < 1) min_stuck = 1;
-	unsigned int min_count = fbc->acceptableConfidence;
+static bool _fbcStallDetect(fbc_t* fbc) {
+	unsigned int minStuck = fbc->acceptableTolerance >> 3;
+	if (minStuck < 1) minStuck = 1;
+	unsigned int minCount = fbc->acceptableConfidence;
 	unsigned int delta = abs(fbc->sense() - _prev);
 
 	if (fbc->output == fbc->neg_deadband || fbc->output == fbc->pos_deadband) {
```

Comment:
seems to mostly work, but if output is 0, it will always detect a stall.  adding `|| fbc->output == 0` to this if statement should solve the problem.